lib/spore/environment.rb

```diff
@@ -17,11 +17,20 @@ def read
     end
 
     def apply
-      each { |k, v| ENV[k] ||= v }
+      each { |k, v| ENV[k] ||= maybe_indirect(v) }
     end
 
     def apply!
-      each { |k, v| ENV[k] = v }
+      each { |k, v| ENV[k] = maybe_indirect(v) }
+    end
+
+    private
+
+    # Indirect values point to other environment variables and start with two
+    # tildes (~~).
```

Comment:
I think I'd prefer to keep it more consistent with `dotenv`. In that library, they do parsing using this substitutions module https://github.com/bkeepers/dotenv/blob/master/lib/dotenv/substitutions/variable.rb